subsys/bluetooth/mesh/net.c

```diff
@@ -893,6 +914,28 @@ void bt_mesh_net_recv(struct net_buf_simple *data, int8_t rssi,
 		net_buf_simple_restore(&buf, &state);
 		bt_mesh_net_relay(&buf, &rx);
 	}
+
+#if IS_ENABLED(CONFIG_BT_MESH_BRG_CFG_SRV)
+	struct pdu_ctx tx_ctx = {
+		.sbuf = &buf,
+		.state = &state,
+		.rx = &rx,
+	};
+
+	/* Bridge the traffic if enabled */
+	if (!bt_mesh_brg_cfg_enable_get()) {
+		return;
+	}
+
+	struct bt_mesh_rpl *rpl = NULL;
```

Comment:
Is there any point of setting = NULL? Point is created default to 0.

Reply:
This is the local variable. It will have garbage value.

Reply:
It needs to be this way, because second param of `rpl_check` is pointer-to-pointer (which we want to set to NULL).